src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/BodyModelBinderProvider.cs

```diff
@@ -52,6 +52,17 @@ public BodyModelBinderProvider(IList<IInputFormatter> formatters, IHttpRequestSt
             _loggerFactory = loggerFactory;
         }
 
+        /// <summary>
+        /// Gets or sets a flag that controls whether the binder should require non-null
+        /// input values to be supplied. Defaults to <c>true</c>.
+        /// </summary>
+        /// <example>
+        /// If <c>true</c>, and an associated <see cref="IInputFormatter"/> supplies <c>null</c>
+        /// as the value to bind (for example, if an incoming POST or PUT request supplies a
+        /// zero-length body), the binder will register a validation error.
+        /// </example>
+        public bool IsBindingRequired { get; set; } = true;
```

Comment:
To set it to `false`, you do:

```csharp
services.AddMvc(options =>
{
    var bodyModelBinderProvider = options.ModelBinderProviders
        .OfType<BodyModelBinderProvider>().Single();
    bodyModelBinderProvider.IsBindingRequired = false;
});
```

Do we definitely want to make it easier than that? Low-discoverability is a feature here, because we don't think most people should mess with this.